src/main/java/pl/arturzgodka/datamodel/FollowerDataModel.java

```diff
@@ -0,0 +1,29 @@
+package pl.arturzgodka.datamodel;
+
+import java.util.List;
+import java.util.Map;
+
+public class FollowerDataModel {
```

Comment:
follower will not be saved into database?

Reply:
Like other class which are used as elements of character class should be saved in database. These elements can change their state or fields values in time so it will be helpful to make easy comparision of state in database with state fetched from API.

Reply:
Need to add @Entity.